gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -274,6 +274,12 @@ nnsconf_loadconf (gboolean force_reload)
     conf.conffile = _strdup_getenv (NNSTREAMER_ENVVAR_CONF_FILE);
   }
 
+  if (conf.conffile == NULL)
+    conf.conffile = g_strdup (NNSTREAMER_DEFAULT_CONF_FILE);
+
+  g_assert (key_file != NULL);
+  g_assert (conf.conffile != NULL);
```

Comment:
IMO, I think this check is not needed.
if `g_strdup()` fails, then as per documentation, `If any call to allocate memory fails, the application is terminated. This also means that there is no need to check if the call succeeded.`

Reply:
This makes sure that ```NNSTREAMER_DEFAULT_CONF_FILE``` is not NULL

Reply:
Right.